cooltools/api/snipping.py

```diff
@@ -82,12 +82,7 @@ def expand_align_features(features_df, flank, resolution, format="bed"):
 
```

Comment:
maybe add a module-wide docstring explaining basic technical details and caveats of snipping
 - what are the basic steps?
 - why pre-fetch entire region-wide matrix instead of fetching individual dense snips
 - are there any resource-related restrictions ? RAM ?